vision-zero

```diff
@@ -187,9 +176,28 @@ def replicateDb(args):
             print("This can take up to 15 minutes.")
         print(" ".join(replicate_runner_command) + " " + " ".join(replicate_command))
         snapshot = open(snapshotFQP, "w")
-        subprocess.run(replicate_runner_command + replicate_command, stdout=snapshot)
+
+        try:
+            result = subprocess.run(
+                replicate_runner_command + replicate_command,
+                stdout=snapshot,
+            )
+        except Exception as e:
+            # rename the corrupted snapshot
+            os.rename(snapshotFQP, snapshotFQP_error)
```

Comment:
renaming the incomplete snapshot file prevents us from entering the doom loop of `replicate-db` sourcing the invalid snapshot file instead of fetching a new one.